### PR TITLE
FAQ - Added how to set up part of the routing

### PR DESCRIPTION
**Pull request by rootux**
# Implements
Updated `FAQ.md` - Added how to set only some of the traffic to go through the VPN and the rest using the normal connection

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -124,3 +124,14 @@ for i in ${userlist[@]};do
     MENU_OPTION=1 CLIENT=$i PASS=1 ./openvpn-install.sh
 done
 ```
+---
+
+**Q:** For my clients - I want to set my internal network to pass through the VPN and the rest to go through my internet?
+
+**A:** You would need to edit the `.ovpn` file. You can edit the template out of which those files are created by editing `/etc/openvpn/client-template.txt` file and adding
```
Comment (randshell) [inline FAQ.md:138]: In my opinion `client-template.txt` deserves its own question. @angristan

**Comment by rootux** [comment]
I agree. Shall I add it?

File: FAQ.md (hunk 1 above)
Comment (angristan) [reply FAQ.md:138]: Agreed!

Comment (rootux) [reply FAQ.md:138]: I agree. Shall I add them?

File: FAQ.md [hunk 2]
```diff
@@ -124,3 +124,21 @@ for i in ${userlist[@]};do
     MENU_OPTION=1 CLIENT=$i PASS=1 ./openvpn-install.sh
 done
 ```
+
+---
+
+**Q:** How do I change the default `.ovpn` file created for future clients?
```
Comment (rootux) [inline FAQ.md:130]: @randomshell @angristan  added general instructions for the `client-template.txt` file

**Review by angristan — APPROVED** [review]

**Pull request merged**